client/src/components/searchResults/rows.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react'
+import { TableRow, TableRowColumn, Avatar, FlatButton } from 'material-ui'
+
+const defaultImage = require('./portrait.png')
+
+const retrieveMedia = media => {
+  const mediaObj = media.reduce((obj, item) => {
+    /* eslint no-param-reassign: ["error", { "props": false }] */
+    obj[item.category] = item
+    return obj
+  }, {})
+  return mediaObj.image ? mediaObj.image.url : defaultImage
+}
```

Comment:
This won't be needed once my changes are merged in

Reply:
Could you explain more about the changes? This handles directly with the payload from the search endpoint, rather than the reducers themselves.